### DISCUSSION TITLE
Expired web/certificate (expired today)

### DISCUSSION THREAD

**Discussion by john2376** [sqlitebrowser/sqlitebrowser#3793] [General]
"The certificate for download.sqlitebrowser.org expired on 2024-11-03."
 (When you go onto downloading the application that warning shows up in browser)
Just downloaded this application per chatGPT's instruction for fixing an unrelated issue, thought I could at least let mods/responsible folks know about this. (mods; feel free to forward this higher up and delete this thread)

**Comment by justinclift** [comment]
Whoops, I'll take a look.  Sounds like I missed renewing a https certificate the other day when I was switching to a different program to manage them.

Thanks for the heads up @john2376. :grin:

**Reply by justinclift** [reply]
Yep, missed that one.  Now fixed.

Sorry for the hassle @john2376. :facepalm:
